### PR TITLE
sqlite

### PR DESCRIPTION
**Pull request by kousun12**
I'd like to have it in a more portable row format so that i can more easily export it to other places, along with some extra metadata

Screen Shot 2023-03-27 at 10 58 50 PM

### CODE REVIEW

File: libai/__init__.py [hunk 1]
```diff
@@ -262,6 +265,55 @@ def _load_chat(chat: str) -> str:
...
+            message["content"],
+            message["role"],
+            completion["model"],
+            query_time,
+            completion["usage"]["prompt_tokens"],
+            path,
+        ),
+    )
+    for choice in completion["choices"]:
+        print(choice)
```
Comment (kortina) [inline libai/__init__.py]: Can You make this a `_dbg` ?

File: libai/__init__.py [hunk 2]
```diff
@@ -272,21 +324,25 @@ def ask(prompt: str, chat=None):
...
+        messages = [system_message]
 
     messages.append(new_message)
 
+    query_time = datetime.datetime.now()
     completion = openai.ChatCompletion.create(
         model=CFG.model,
         messages=messages,
     )
+    response_time = datetime.datetime.now()
```
Comment (kortina) [inline libai/__init__.py:353]: NIce

**Review by kortina — APPROVED** [review]
Looks good. So this will *always* save to both markdown and sqlite, correct?

One nit about print vs dbg

**Comment by kousun12** [comment]
ya always saves to both as is. but we can put behind an option too if you think that makes more sense

File: libai/__init__.py (hunk 1 above)
Comment (kousun12) [reply libai/__init__.py]: ah this was just a random bug fixing print, unintentional

**Comment by kortina** [comment]
Are you able to merge this? I think I granted you permissions?

image

**Comment by kousun12** [comment]
ya i have merge now

**Pull request merged**